datasets/comonscreens.yaml

```diff
@@ -0,0 +1,28 @@
+Name: Common Screens
+Description: A corpus of web screenshot and metadata data composed of over 70 million websites.
+Documentation: https://commonscreens.com/?page_id=1492
+Contact: admin@commonscreens.com
+ManagedBy: "[Common Screens](https://commonscreens.com/)"
+UpdateFrequency: Monthly
+Tags:
+  - aws-pds
+  - encyclopedic
+  - natural language processing
+  - internet
+License:  Creative Commons license

```

Comment:
Apologies @asansanwal but please define *a* CC license. For example, one from https://creativecommons.org/about/cclicenses/. https://creativecommons.org/licenses/by/4.0/ is a pretty typical one.